docs/advanced_topics/images/renditions.rst

```diff
@@ -0,0 +1,28 @@
+.. _image_renditions:
+
+Generating renditions in Python
+=====================================
+
+Rendered versions of original images generated by the Wagtail ``{% image %}`` template tag are called "renditions,"
+and are stored as new image files in the site's ``[media]/images`` directory on the first invocation.
+
+Image renditions can also be generated dynamically from Python via the native ``get_rendition()`` method, e.g.:
+
+ .. code-block:: python
+
+    newimage = myimage.get_rendition('fill-300x150|jpegquality-60')
+
+Argument options are identical to arguments accepted by the ``{% image %}`` template tag, and should be separated
```

Comment:
> Argument options are identical to arguments accepted by the ``{% image %}`` template tag

I think it should be clearer that we mean the filter spec here, rather than the other keyword arguments that the template tag can take.